### PR TITLE
rtmp-services: Add GameTips.tv

### PR DESCRIPTION
**Pull request by h-o-sein**
This service is for Iranian Gamers.

### CODE REVIEW

**Comment by derrod** [comment]
Please check the commit guidelines, the commit message should be formatted as such: `rtmp-services: Add GameTips.tv`. Also `common` has to be omitted.

**Comment by h-o-sein** [comment]
> Please check the commit guidelines, the commit message should be formatted as such: `rtmp-services: Add GameTips.tv`. Also `common` has to be omitted.

Thankful
I corrected it

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -779,6 +779,16 @@
                 "bframes": 0
             }
         },
+        {
+          "name": "GameTips.TV - Service Stream Game Iran",
+          "common": true,
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json]: As this is not a common service, this line should be omitted.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -779,6 +779,16 @@
                 "bframes": 0
             }
         },
+        {
+          "name": "GameTips.TV - Service Stream Game Iran",
```
Comment (dodgepong) [inline plugins/rtmp-services/data/services.json]: Does the name need to be this long? I would think that `GameTips.TV` should be sufficient.

Comment (h-o-sein) [reply plugins/rtmp-services/data/services.json]: Give it a change: GameTips.TV

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (h-o-sein) [reply plugins/rtmp-services/data/services.json]: This service is special for Iran

**Comment by DDRBoxman** [comment]
Will be merged in 8b6b8f33de47c14f563073ed4a235e72a710f33e

**Pull request closed without merging**